src/components/QueryBuilder.vue

```diff
@@ -85,6 +87,20 @@ export default Vue.extend( {
 			this.$store.dispatch( 'setErrors', validationResult.formErrors );
 			this.fieldErrors = validationResult.fieldErrors;
 		},
+		validateForLimitedSupport( selectedProperty: SearchResult ): void {
+			this.limitedSupport = false;
+			this.fieldErrors.property = null;
+			this.selectedPropertyValueRelation = PropertyValueRelation.Matching;
+			const allowedDatatypes = [ 'external-id', 'string' ];
+			if ( selectedProperty && allowedDatatypes.indexOf( selectedProperty.datatype ) === -1 ) {
```

Comment:
This can use [Array.prototype.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

Reply:
Done.